### DISCUSSION TITLE
Planejarmento de aulas Frontend : como criar um bom plano de aulas para ensinar HTML, CSS, JS Basico.

### DISCUSSION THREAD

**Discussion by Gpaula95** [frontendbr/forum#1234] [Discussão]
Olá pessoal, eu sou desenvolvedor frontend a cerca de 1 ano e meio, ajudo em alguns projetos voluntariamente na faculdade, e um deles é um curso de frontend inicialmente basico, não tenho experiencia com aulas, portanto tenho duvidas de como montar um bom plano de aula para ensinar outros graduandos HTML, CSS, JS. e quais pré requisitos eu preciso colocar para a matricula do curso.  Aceito dicas  e artigos para que eu consiga ensinar de forma efetiva e cativar esses estudantes a se aventurar na carreira de frontend futuramente. 

ps. é meu primeiro post, caso eu tenha estruturado de forma errada, por favor me corrijam.

**Comment by victornery** [comment]
Fala @Gpaula95, tranquilo cara? 

Eu sou um grande fã da educação e tenho um sonho de mudar a vida de pessoas com desenvolvimento web. Para isso, é preciso saber cativar bem os alunos e incentivar eles a prosseguir, continuar no aprendizado contínuo. 

Eu te recomendo a ver como outros desenvolvedores organizam grades de ensino, como a @FernandaBernardo, que tem um curso de Desenvolvimento Web na Udemy. 

A minha recomendação pessoal é começar com HTML e CSS, talvez os assuste menos no início. Deixe-os claro da existência do JavaScript e vá os introduzindo aos poucos. Como pré-requisito, acredito que você possa botar como Recomendavel uma noção de Lógica de Programação.

Além disso, tenta criar alguns wireframes pra passar projetos durante o curso.

Espero ter te ajudado.

**Comment by Gpaula95** [comment]
Eai @victornery, tudo tranquilo e por ai ? muito legal !

Eu sou um grande fã de educação e compartilhamento de conhecimento também, tenho uma grande vontade de palestrar, e ensinar, retribuindo tudo que já aprendi e formando e fortalecendo o frontend !

A @FernandaBernardo  é um exemplo com certeza, sou um fã das palestras dela a didatica dela e sem igual !! 
Muito obrigado me ajudou sim, vou iniciar meu plano de aula aqui, dou um feedback quando estiver dando o curso ! 
abraço !

**Comment by felipefialho** [comment]
@fdaciuk @willianjusten

**Comment by lucasferreiralimax** [comment]
@Gpaula95 
Também tento imergir novos padawans no universo web, uma comunidade e plataforma muito completa e boa para ir ensinando code live na veia é a [freecodecamp](https://www.freecodecamp.org) não sei se você já ouviu falar do projeto.

É uma plataforma muito boa para se ter uma base para começar a lesionar sobre as linguagens web, vale a pena colaborar e ajudar assim a plataforma vai melhorando e evoluindo cada vez mais.

A grade curricular que está disponível no [freecodecamp](https://www.freecodecamp.org) vai desde o básico ao tratamento avançado de dados, onde cada um vai progredindo conforme o próprio aprendizado.
